docs/integrations/new-editor.md

```diff
@@ -128,8 +128,9 @@ The currently available settings for `InitializationOptions` are listed below.
       "renameFileThreshold": number,
       "slowTaskProvider": boolean,
       "statusBarProvider": "on" | "off" | "show-message" | "log-message",
-      "treeViewProvider": boolean
-      "openNewWindowProvider": boolean
+      "treeViewProvider": boolean,
+      "openNewWindowProvider": boolean,
+      "copyWorksheetOutputProvider": boolean
```

Comment:
_Extremely nit picky_ 😬 but I put can we move the new value up to be in alphabetical order? It helps when easily scanning all the init options in the docs.

Reply:
Not that nitpicky really :sweat_smile: Moved it up and also fixed the order for `openNewWindowProvider`

Reply:
🙏🏼